fhir-smart/src/main/java/com/ibm/fhir/smart/AuthzPolicyEnforcementPersistenceInterceptor.java

```diff
@@ -539,10 +537,12 @@ private String getAccessToken() throws FHIRPersistenceInterceptorException {
 
         String patientId = claim.asString();
         if (patientId == null) {
-            log.fine("Found patient_id claim was expected to be a string but is not; processing as a list");
             return claim.asList(String.class);
         }
 
-        return Collections.singletonList(patientId);
+        return Stream.of(patientId.split(" "))
```

Comment:
Should we split on \s+ here?

Reply:
Good question.  I think we're ok with supporting just space-delimited values for now, but do let me know if you feel we should change it.